kong/plugins/grpc-web/handler.lua

```diff
@@ -0,0 +1,91 @@
+-- Copyright (c) Kong Inc. 2020
+
+local deco = require "kong.plugins.grpc-web.deco"
+
+local ngx = ngx
+local kong = kong
+
+local string_format = string.format
+local ngx_req_get_method = ngx.req.get_method
+local kong_request_get_path = kong.request.get_path
+local kong_request_get_header = kong.request.get_header
+local kong_request_get_raw_body = kong.request.get_raw_body
+local kong_response_exit = kong.response.exit
+local kong_response_set_header = kong.response.set_header
+local kong_service_request_set_header = kong.service.request.set_header
+local kong_service_request_set_raw_body = kong.service.request.set_raw_body
+
+
+local grpc_web = {
+  PRIORITY = 1,
+  VERSION = '0.1.0',
+}
+
+
+function grpc_web:access(conf)
+  kong_response_set_header("Access-Control-Allow-Origin", "*")
+
+  if ngx_req_get_method() == "OPTIONS" then
```

Comment:
Really not a huge issue, but there has been consensus that newer plugin code should utilize the PDK functions as much as possible. Which in this case, would be [kong.request.get_method()](https://docs.konghq.com/2.0.x/pdk/kong.request/#kongrequestget_method).

Reply:
done